cli/azd/pkg/project/framework_service_python.go

```diff
@@ -130,12 +130,7 @@ func (pp *pythonProject) Package(
 	if err := buildForZip(
 		packageSource,
 		packageDest,
-		buildForZipOptions{
-			excludeConditions: []excludeDirEntryCondition{
-				excludeVirtualEnv,
-				excludePyCache,
-			},
-		}); err != nil {
+		buildForZipOptions{}); err != nil {
```

Comment:
These exclusions where moved to project_utils, because they are now dependent on the existance of .zipignore